doc/source/building/index.rst

```diff
@@ -290,6 +290,23 @@ the SciPy repository::
       cd scipy
       git submodule update --init
 
+
+.. tip::
+
+    Many of the steps described below can now be accomplished automatically
+    with commands which execute tasks in SciPy's Pixi workspace,
+    like ``pixi run build``.
+    To use this workspace, `install Pixi <https://pixi.sh/latest/installation/>`__
```

Comment:
Probably too idiosyncratic to mention in the docs, but two issues I've had with `pixi` in an HPC context:

1) It mutates my shell profile (i.e., `~/.tcshrc`) by default, and that can mess around with `module` settings and stuff, which cascades to all kinds of other issues like causing HPC CI (https://gitlab.com/ecp-ci/jacamar-ci) to fail because it uses my mutated shell profile.
2) The install instructions linked don't work because the network is blocked; that's a bit easier to work around--I can vendor the install bash script locally and then it works.

Reply:
And agreed to not mention that here.